sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsFilterForm.java

```diff
@@ -130,6 +142,64 @@ public void addMoreFilters(CustomLayout moreFiltersContainer) {
 
 		moreFiltersContainer.addComponent(buildWeekAndDateFilter(EventCriteria.DateType.EVENT), EVENT_WEEK_AND_DATE_FILTER);
 		moreFiltersContainer.addComponent(buildWeekAndDateFilter(EventCriteria.DateType.ACTION), ACTION_WEEK_AND_DATE_FILTER);
+
+		ComboBox facilityTypeGroup = new ComboBox();
+		facilityTypeGroup.setId(FACILITY_TYPE_GROUP_FILTER);
+		facilityTypeGroup.setInputPrompt(I18nProperties.getCaption(Captions.Facility_typeGroup));
```

Comment:
Please add TYPE_GROUP as a constant to FacilityDto (it's missing there) and use I18nProperties.getPrefixCaption